salesagility/SuiteCRM
# Subpanels Templates

Moving sub panel hard coded content to tpls.

## Description
This effects:
- elements that use the get_form_header() function (include/utils/layout_utils.php)
- elements that use the SugarWidgetTabs class  (include/tabs.php)
- elements that use the SubPanelTiles class (include/SubPanel/SubPanelTiles.php)
- Dashboard
- Dashlets
- Subpanels

## Motivation and Context
In order to customise the the html per theme, we need to have a tpl we can override. My intention is to move to using the new bootstrap panels that are in the detail and edit views.

## How To Test This
- Check any page that has subpanels or dashlets.
- Test each theme to ensure they behave the same way.

## Types of changes
- Feature

### Final checklist
- [] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [] My change requires a change to the documentation.
